app/services/gmail/delete.py

```
@@ -124,9 +146,47 @@ def process_message(request_id, response, exception) -> None:
+                        .get(
+                            userId="me",
+                            id=msg_id,
+                            format="metadata",
+                            metadataHeaders=["From", "Subject", "Date"],
+                        )
+                        .execute()
+                    )
+                    process_message_response(response)
+                    processed += 1
+                    return
+                except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)